src/pages/index.astro

```diff
@@ -16,59 +17,214 @@ const options: ISourceOptions = background_config
-                <p>Find our documentation at <a href="https://docs.khoj.dev">docs.khoj.dev</a>.</p>
-                <p>
-                    If you want to reach out, give feedback, or have a question, <a href="https://discord.gg/BDgyabRM6e">join us on Discord</a> or consult us on <a href="https://github.com/khoj-ai/khoj/issues/">GitHub</a>.
-                </p>
+                <canvas id="scene"></canvas>
+
+                <div id="text">
+                    <p>
+                        Welcome to the official Khoj blog. We'll document our thoughts here about personal AI, product development, and the future of work.
+                    </p>
+                    <p>
+                        Start experimenting with your own personal AI at <a href="https://khoj.dev">khoj.dev</a>.
+                    </p>
+                    <p>We're making core consumer AI applications to ensure the future of digital services is trustworthy, transparent, and deeply useful. The world is going to change a lot in the coming decade, and we want to make this shift happen responsibly and delightfully.</p>
+                    <p>Find our documentation at <a href="https://docs.khoj.dev">docs.khoj.dev</a>.</p>
+                    <p>
+                        If you want to reach out, give feedback, or have a question, <a href="https://discord.gg/BDgyabRM6e">join us on Discord</a> or consult us on <a href="https://github.com/khoj-ai/khoj/issues/">GitHub</a>.
+                    </p>
+                </div>
             </main>
-            <Particles params={{}} id="tsparticles" options={options} init="particlesInit" />
+            <Particles id="tsparticles" options={options} init="particlesInit" />
         </div>
         <Footer />
     </body>
 </html>
 <style>
+    body{
+        position: relative;
+    }
      main {
-        color: black;
         margin: 30px auto;
-    }
-    main a{
-        color: blue;
+        display: flex;
+        flex-direction: column;
+        justify-content:center;
+        align-items: center;
     }
     #content{
         position: relative;
+    } 
+
+    canvas{
+        z-index:10;
+    }
+    #text{
+        display: flex;
+        flex-direction: column;
+        align-items: left;
     }
+</style>
+<script>
+    document.addEventListener('DOMContentLoaded', () => {
+        const rootStyles = getComputedStyle(document.documentElement);
+        const lightBackgroundColor = rootStyles.getPropertyValue('--background-color-light').trim();
+        const darkBackgroundColor = rootStyles.getPropertyValue('--background-color-dark').trim();
+        const theme = localStorage.getItem('theme');
+        if (theme === 'dark') {
+            document.body.style.backgroundColor = darkBackgroundColor;
+        } else {
+            document.body.style.backgroundColor = lightBackgroundColor;
+        }
+    });
```

Comment:
Can you also do this in ThemeIcon.astro? 🙂 Last change, thanks for taking care of everything!